### ISSUE TITLE
simplify install process, use current appengine standards (gcloud vs appcfg, and app.yaml config)

### ISSUE THREAD

**Issue by ncouture** [googleads/googleads-python-lib#248]
Throwing this in the air again as resolved by PR #222:

 - remove extranuous installation instructions
 - add requirements.txt
 - add appengine_config.py, use of lib/ dir for 3rd party Python libs
 - declare default service name in app.yaml
 - ask for installation of the Google Cloud SDK instead of App Engine SDK;
   + install steps use `gcloud` over  `appcfg`

Could someone review?

**Comment by msaniscalchi** [comment]
I did run through the changes introduced in the PR some time ago, but I recall that there were some issues I discovered. Further digging on this topic got pushed off in favor of some higher priority work for the library, sorry about not following up on that sooner!

I'll take another pass at this for the upcoming AdWords release. This is a duplicate, but I'll keep it open with a prioritization of P2 for additional visibility.
